site/app/controllers/UserProfileController.php

```diff
@@ -149,6 +149,9 @@ public function changeProfilePhoto() {
             // Save image for user
             $result = $user->setDisplayImage($extension, $_FILES['user_image']['tmp_name']);
             $display_image = $user->getDisplayImage();
+            if ($result === \app\models\User::PROFILE_IMG_QUOTA_EXHAUSTED) {
```

Comment:
You should be able to import the User model with a "use" statement and then this could be shortened to just User::PROFILE_IMG_QUOTE_EXHAUSTED